logviewer/src/main/java/org/togetherjava/logwatcher/DatabaseProvider.java

```diff
@@ -0,0 +1,66 @@
+package org.togetherjava.logwatcher;
+
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.FatalBeanException;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.togetherjava.tjbot.db.Database;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.SQLException;
+import java.util.concurrent.atomic.AtomicReference;
+
+@Configuration
+public class DatabaseProvider {
+
+    private static final AtomicReference<Database> AtomDb = new AtomicReference<>();
```

Comment:
Name conflicts style:
```suggestion
    private static final AtomicReference<Database> ATOM_DB = new AtomicReference<>();
```